backend/config/environments/test.rb

```diff
@@ -43,7 +43,7 @@
   # Tell Action Mailer not to deliver emails to the real world.
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
-  # config.action_mailer.delivery_method = :test
+  config.action_mailer.delivery_method = :test
```

Comment:
Note to self @chrisbendel comment this out when Iris is ready to test real emails